qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -272,29 +255,72 @@ def defaults(self, refresh: bool = False) -> Optional[PulseDefaults]:
 
         return self._defaults
 
-    def jobs_limit(self) -> BackendJobsLimit:
-        """Return the jobs limit for the backend.
+    def job_limit(self) -> BackendJobLimit:
+        """Return job limit for the backend.
+
+        The job limit information may include the current number of
+        jobs running on the backend at this time and the maximum number
+        of concurrent jobs a user could submit to the backend at a time.
 
         Note:
-            If `BackendJobsLimit.maximum_jobs = -1`, then there
-            are no limits to the maximum number of jobs that could
-            be submitted on the backend.
+            The job limit information for the backend is given for
+            this account (i.e. a specific backend with a specific
+            provider).
+
+            If the method call was successful, you can inspect the job
+            limit for the backend by accessing the ``maximum_jobs``
+            and ``running_jobs`` attributes of the ``BackendJobLimit``
+            class returned.
+
+            For example:
+                backend_job_limit = IBMQBackend.job_limit()
```

Comment:
It's probably better to do `backend.job_limit()` so users don't confuse it for a class method.